platform/src/pages/account/creation/index.jsx

```diff
@@ -1,9 +1,25 @@
-import React from 'react';
+import React, { useState } from 'react';
 import AccountPageLayout from '../../../common/components/Account/Layout';
 import GoogleLogo from '@/icons/Common/google_logo.svg';
```

Comment:
@FaithDaka let's use the absolute imports throughout eg components are `@/components/*`

Reply:
Thanks @FaithDaka 
The absolute import for the Google icon looks interesting: `@/icons`....could it be a typo?